person.rb

```diff
@@ -19,6 +20,11 @@ def correct_name
     name
   end
 
+  def add_rental(rental)
+    @rentals << rental unless @rentals.include?(rental)
```

Comment:
- [ ] According to the project requirements and relationship between the `Person` and `Book` calsses using the intermediate class `Rental`, I think it would be better if your update your `#add_rental` method implementation as it should include a `date` and a `book` as parameters and also the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association i.e.

```ruby
   require_relative 'rental'


  ...
  def add_rental(date, book)
    `Rental.new(date, book, self)`
  end
```